glide-core/src/client/value_conversion.rs

```diff
@@ -201,7 +201,7 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
             Some(ExpectedReturnType::Map)
         }
         b"INCRBYFLOAT" | b"HINCRBYFLOAT" => Some(ExpectedReturnType::Double),
-        b"HEXISTS" | b"EXPIRE" | b"EXPIREAT" | b"PEXPIRE" | b"PEXPIREAT" => {
+        b"HEXISTS" | b"EXPIRE" | b"SISMEMBER" | b"EXPIREAT" | b"PEXPIRE" | b"PEXPIREAT" => {
```

Comment:
minor: why not adding it at the end? it seems unrelated in the middle of expire commands